dinstaller-cli/src/main.rs

```diff
@@ -24,6 +24,21 @@ struct Cli {
 fn probe() {
     let client = manager::ManagerClient::new(dinstaller_lib::connection().unwrap()).unwrap();
     client.probe().unwrap();
+    show_progress(client)
+}
+
+fn install() {
```

Comment:
If you return a `Result`, you can use the `?` operator.

```rust
fn install() -> Result<(), Box<dyn Error>> {
  let client = manager::ManagerClient::new(dinstaller_lib::connection()?)?;
  # ...
}
```

But then you should use `install().unwrap()` below...

Reply:
well, it will make sense if we have some generic handler for errors below. If we want to use that approach than it makes sense and also probe should be changed same way.